conda_pack/scripts/windows/deactivate.bat

```diff
@@ -40,6 +40,6 @@
 :filterPath
   @IF "%~1" == "" @GOTO :filterOut
   @FOR %%j IN ("%TARGETS:;=";"%") DO @IF /i "%~1" == "%%~j" @GOTO :filterOut

```

Comment:
The failures were due to the fact that the IF/THEN pair was making this line too long. I opted to avoid this by always adding the semicolon even for an empty path, then stripping off that leading semicolon later.